lib/tasks/local_dev.rake

```diff
@@ -56,7 +56,7 @@ task copy_feature_flags_from_production: :environment do
   puts 'Synchronising feature flags with production...'
 
   FeatureFlag::FEATURES.each_key do |f|
-    if flags.dig(f, 'active')
+    if flags.dig(f, 'active') && f.to_s != 'send_request_data_to_bigquery'
```

Comment:
wonder if instead of specifying big query, we ignore `variant` type features instead so if we add more of those they aren't pulled in later?
```
if flags.dig(f, 'active') && flags.dig(f, 'type') != 'variant'
```